.gitignore

```diff
@@ -15,3 +15,6 @@ graphql.config.json
 graphql.schema.json
 coverage
 /storybook-static
+
+# macOS
+.DS_Store
```

Comment:
This should be in the global git configuration, it's not specific to the project environement